subprojects/docs/src/samples/userguide/buildlifecycle/projectEvaluateEvents/kotlin/projectA.gradle.kts

```diff
@@ -0,0 +1 @@
+setProperty("hasTests", true)
```

Comment:
:x: should be `extra["hasTests"] = true` to match the root project using extra properties explicitely